roles/wordpress-setup/templates/wordpress-site.conf.j2

```diff
@@ -7,7 +7,7 @@ server {
   listen 80;
   {% endif %}
 
-  server_name  {{ item.value.site_hosts | join(' ') }};
+  server_name  {% for host in item.value.site_hosts %} {{ host }} {% if item.value.multisite.subdomains %} *.{{ host }} {% endif %} {% endfor %};
```

Comment:
@louim I haven't been told why the wildcard in `site_hosts` fails, and I'm afraid I haven't looked into it. Here is an [example of it failing](https://discourse.roots.io/t/multisite-provisioning-failure/4328/5), but here's an example of a [trellis fork where it doesn't fail](https://github.com/roots/trellis/issues/266#issuecomment-120727239).